org.genivi.DLTViewer.yml

```diff
@@ -0,0 +1,33 @@
+app-id: org.genivi.DLTViewer
+runtime: org.kde.Platform
+runtime-version: '5.15'
+sdk: org.kde.Sdk
+command: dlt-viewer
+finish-args:
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --share=ipc
+  - --device=all
+  - --share=network
+  - --persist=.dlt
+  - --env=LD_LIBRARY_PATH=/app/usr/lib/x86_64-linux-gnu
+cleanup:
+  - /usr/include
+  - /usr/share/dlt-viewer/plugins/libdummy*.so
```

Comment:
```suggestion
  - /include
  - /share/dlt-viewer/plugins/libdummy*.so
```

Reply:
This won't work because the application installs everything inside $PREFIX/usr. The /usr is hardcoded. I tried to remove it but I didn't know how to handle the qdlt install path which is $QT_INSTALL_LIBS and has /usr hardcoded too.